app/views/foreman_statistics/trends/index.html.erb

```diff
@@ -33,7 +24,7 @@
   </tbody>
 </table>
 <%= will_paginate_with_info @trends %>
-<% unless ForemanStatistics::TrendCounter.unconfigured? %>
+<% if (last_counter = ForemanStatistics::TrendCounter.order(:created_at).last) %>
```

Comment:
not crucial but you can use AR here to pull the maximum directly without loading the whole record:
```suggestion
<% if (last_counter = ForemanStatistics::TrendCounter.maximum(:created_at)) %>
```